td/tracking/templates/tracking/project_list.html

```diff
@@ -26,9 +26,8 @@
                 {% endblocktrans %}
             {% else %}
             <p>
-                {% url "account_login" as login_url %}
                 {% blocktrans %}
-                    <a href="{{ login_url }}" class="btn btn-default">Log In to Enter Project or Event</a>
+                    <a href="/account/login/?next=/tracking/" class="btn btn-default">Log In to Enter Project or Event</a>
                 {% endblocktrans %}
```

Comment:
Let's not hard code urls.

```
{{ login_url }}?next={{ tracking_url }}
```

should work. Just reverse the `tracking_url` with a `{% url %}` tag above the `{% blocktrans %}` block.